examples/post_training_quantization/tensorflow/mobilenet_v2/requirements.txt

```
@@ -1,4 +1,4 @@
 tensorflow~=2.12.0
 tensorflow-datasets
 tqdm
-openvino-dev==2023.0.1
+openvino-dev==2023.1
```

```suggestion
openvino-dev==2023.0.2
```
take into account test_examples/92/ results

---

Build - 93